tests/test_unit/approval_client/client.py

```diff
@@ -795,6 +796,19 @@ def return_values(*args, **kwargs):
 
             return read_list_cls(existing_resources[resource_cls.__name__], cognite_client=client)
 
+        def return_data_models(
+            ids: DataModelIdentifier | Sequence[DataModelIdentifier], inline_views: bool = False
```

Comment:
This is the same antipattern I commented on somewhere else.  I'm not going to block this pr, but I would like to see us not doing this are a regular practice.